anoncreds/src/data_types/anoncreds/cred_def.rs

```diff
@@ -33,12 +33,15 @@ pub struct CredentialDefinitionData {
 }
 
 #[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct CredentialDefinition {
     pub schema_id: SchemaId,
     #[serde(rename = "type")]
     pub signature_type: SignatureType,
     pub tag: String,
     pub value: CredentialDefinitionData,
+    #[serde(rename = "issuerId")]
```

Comment:
This is not needed right, as you also have camelCase rename on the struct?

Reply:
yes, you are correct. I always do both, no clue why.